CHANGELOG.md

```diff
@@ -233,6 +233,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - `seed_everything` now fails when an invalid seed value is passed instead of selecting a random seed ([#8787](https://github.com/PyTorchLightning/pytorch-lightning/pull/8787))
 
 
+- Directly call TrainingTypePlugin collective APIs instead of going through the Accelerator ([#9426](https://github.com/PyTorchLightning/pytorch-lightning/pull/9426))
```

Comment:
```suggestion
- Directly call `TrainingTypePlugin` collective APIs instead of going through the Accelerator ([#9677](https://github.com/PyTorchLightning/pytorch-lightning/pull/9677))
```